pyroomacoustics/transform/stft.py

```diff
@@ -31,7 +31,7 @@ class STFT(object):
     transform (optional) : str
         which FFT package to use: 'numpy' (default), 'pyfftw', or 'mkl'
     streaming (optional) : bool
-        whether (True) or not (False, default) to "stitch" samples between 
+        whether (True, default) or not (False) to "stitch" samples between
```

Comment:
So what does making `streaming=True` the default mean practically ? Can we still just create the object and dump a large signal to change all at once ? I guess the answer is yes but, please confirm.

Reply:
Practically it means that consecutive calls of ``analysis``/``synthesis`` will by default stitch the consecutive frames (even if a particular call of ``analysis`` consists of multiple frames). ``streaming=False`` implies that consecutive calls of ``analysis``/``synthesis`` will not stitch the chunks together. I just noticed a problem with the ``stft_block_processing.py`` example where ``streaming`` was not set to ``True`` but it should have been. In any case, it's up for discussion whether we want this stitching to be done by default or not. I could also manually set ``streaming=True`` for the constructor in the ``stft_block_processing.py`` example.